commands/__tests__/auth.test.ts

```diff
@@ -0,0 +1,57 @@
+// @ts-nocheck
+import yargs from 'yargs';
+import { addConfigOptions, addTestingOptions } from '../../lib/commonOpts';
+
+jest.mock('yargs');
+jest.mock('../../lib/commonOpts');
+
+// Import this last so mocks apply
+import authCommand from '../auth';
+
+describe('commands/auth', () => {
+  describe('command', () => {
+    it('should have the correct command structure', () => {
+      expect(authCommand.command).toEqual('auth [type] [--account]');
+    });
+  });
+
+  describe('describe', () => {
+    it('should provide a description', () => {
+      expect(authCommand.describe).toBeDefined();
+    });
+  });
+
+  describe('builder', () => {
+    it('should support the correct positional arguments', () => {
+      authCommand.builder(yargs);
+
+      expect(yargs.positional).toHaveBeenCalledTimes(1);
+      expect(yargs.positional).toHaveBeenCalledWith('type', {
```

Comment:
Similar to my point about command description, I'm also on the fence about whether or not it's valuable for us to be verifying the exact description copy for options and positionals. I'm considering updating this to only validate things like name and type for the options.

Reply:
That makes sense to me